app/routes/users/components/UserProfile.js

```diff
@@ -47,20 +49,52 @@ type UpcomingEventsProps = {
   upcomingEvents: Array<Event>
 };
 
-const GroupPill = ({ group }: { group: Group }) => (
-  <Pill key={group.id} style={{ margin: '5px' }}>
+const GroupPill = ({ group }: { group: Group }) =>
+  group.showBadge && (
+    <Pill key={group.id} style={{ margin: '5px' }}>
+      {group.name}
+    </Pill>
+  );
+
+const BadgeTooltip = ({
+  group,
+  start,
+  end
+}: {
+  group: Group,
+  start: Dateish,
+  end: ?Dateish
+}) => (
+  <div>
     {group.name}
-  </Pill>
+    {' ('}
```

Comment:
This looks kinda funky, no other way to format this code?

Reply:
:+1: 
10/10 unreadable code